index.html

```diff
@@ -47,9 +47,9 @@
         for title in response.css('h2.entry-title'):
             yield {'title': title.css('a ::text').extract_first()}
```

Comment:
I think we can start using it in next release, and update Scrapy docs to use it as well. To do so we may increase parsel version in install_requires.